includes/class-wc-gateway-stripe.php

```diff
@@ -404,6 +404,32 @@ public function payment_scripts() {
 			return;
 		}
 
+		if (
+			is_product()
+			&& ! in_array(
+				'product',
+				$this->get_option( 'payment_request_button_locations', [ 'product', 'cart' ] ),
+				true
+			)
+		) {
+			if ( apply_filters( 'wc_stripe_remove_scripts_on_product_page_when_prbs_disabled', false ) ) {
```

Comment:
These filter names are very much up for discussion. I'm trying to get their function across in the name, but the names are pretty long. Not sure if they can be shortened without losing some of the meaning.

I'd be happy to get some feedback on the naming for both filters.

Reply:
I changed the filter names to `wc_stripe_load_scripts_on_cart_page_when_prbs_disabled` and `wc_stripe_load_scripts_on_product_page_when_prbs_disabled` to better reflect how they're used in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2110/commits/a152af75ed9d260b36f6ee6637c6457038eb498d